packages/components/src/index.js

```diff
@@ -50,6 +50,7 @@ export { default as TableCard } from './table';
 export { default as EmptyTable } from './table/empty';
 export { default as TablePlaceholder } from './table/placeholder';
 export { default as TableSummary } from './table/summary';
+export { TableSummaryPlaceholder } from './table/summary';
```

Comment:
This can be combined with the line above:

```suggestion
export { default as TableSummary, TableSummaryPlaceholder } from './table/summary';
```

Reply:
😁 I tried following most of other `export` lines in this file.
Anw, addressed here a2689c5e62eb6ece2e0cf85934e481cc0d0d04f8